src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx

```diff
@@ -10,46 +18,70 @@ export interface ArtworkDetailsAboutTheWorkFromPartnerProps {
   artwork: ArtworkDetailsAboutTheWorkFromPartner_artwork
 }
 
-export const ArtworkDetailsAboutTheWorkFromPartner: React.SFC<
+const InitialsHolder = styled(Box)`
```

Comment:
Not sure it was the best way to accomplish it - but this makes the avatar background when there is not image on partner profile.